crates/runner/src/lib.rs

```diff
@@ -10,10 +10,93 @@ use wasi_common::sync::WasiCtxBuilder;
 use wasi_common::WasiCtx;
 use wasmtime::{Config, Engine, Linker, Module, OptLevel, Store};
 
+pub struct Builder {
+    /// The JS source.
+    input: PathBuf,
+    /// Root path. Used resolve the absolute path of the JS source.
+    root: PathBuf,
+    /// `javy` binary path.
+    bin_path: String,
+    /// The path to the wit file.
+    wit: Option<PathBuf>,
+    /// The name of the wit world.
+    world: Option<String>,
+    /// The logger capacity, in bytes.
+    capacity: usize,
```

Comment:
I think we can get rid of capacity. Initially I added it because I wanted to test a change for what should happen if the host-side buffer size is exceeded but IIRC we never altered anything in Javy so I think we're just always setting this to `usize::MAX`.